storage/client/client.go

```diff
@@ -68,14 +68,35 @@ func translateTokenType(tokenType common.TokenType) apiTypes.EvmTokenType {
 	}
 }
 
+// The apiTypes Layers may be named differently from Nexus-internal Layers
+// to make the api more ergonomic. Currently the only differing layer is
```

Comment:
If we make the API less ergonomic (that's arguable anyway), does that make this PR much simpler? I'm sure the frontend can adapt.

Reply:
Hm not by much, the bulk of this PR was for translating `pontusx_test` in the sdk to `pontusx` in Nexus. I updated the PR to now just use `pontusx_test` everywhere, so now the `pontusx_test` data is served under `/pontusxtest/`, see comment below.